ckan/setup/server_start.sh

```diff
@@ -10,7 +10,7 @@ DIR="$(dirname "${BASH_SOURCE[0]}")"
 # Threads RAM requirement unknown at this time
 # exec newrelic-admin run-program gunicorn -c "$DIR/gunicorn.conf.py" --worker-class gevent --paste $CKAN_INI "$@"
 if [[ "$CKAN_SITE_URL" = "http://ckan:5000" ]]; then
-  exec newrelic-admin run-program gunicorn "wsgi:application" --config "$DIR/gunicorn.conf.py" -b "0.0.0.0:$PORT" --chdir $DIR  --timeout 120 --workers 2
+  exec pipenv run newrelic-admin run-program gunicorn "wsgi:application" --config "$DIR/gunicorn.conf.py" -b "0.0.0.0:$PORT" --chdir $DIR  --timeout 120 --workers 2
 else
   exec newrelic-admin run-program gunicorn "wsgi:application" --config "$DIR/gunicorn.conf.py" -b "0.0.0.0:$PORT" --chdir $DIR  --timeout 120 --worker-class gevent --workers 4 --threads 4 --forwarded-allow-ips='*'
```

Comment:
Equivalent pipenv run needed here possibly; or maybe not? would require cloud.gov testing.

Reply:
Okay, I'll look into this.